test/test-tcp-close-reset.c

```diff
@@ -62,9 +68,18 @@ static void do_write(uv_tcp_t* handle) {
 
 
 static void do_close(uv_tcp_t* handle) {
+  uv_os_fd_t fd;
+  int r;
+
   if (shutdown_before_close == 1) {
     ASSERT(0 == uv_shutdown(&shutdown_req, (uv_stream_t*) handle, shutdown_cb));
     ASSERT(UV_EINVAL == uv_tcp_close_reset(handle, close_cb));
+  } else if (shutdown_before_close == 2) {
+    r = uv_fileno((const uv_handle_t*) handle, &fd);
+    ASSERT(r == 0);
+    ASSERT(fd != INVALID_FD);
+    ASSERT(0 == shutdown(fd, SHUT_RDWR));
+    ASSERT(0 == uv_tcp_close_reset(handle, close_cb));
```

Comment:
Nit: use the newer ASSERT_* macros here and in the rest of the changes.
```suggestion
    ASSERT_EQ(r, 0);
    ASSERT_NE(fd, INVALID_FD);
    ASSERT_EQ(0, shutdown(fd, SHUT_RDWR));
    ASSERT_EQ(0, uv_tcp_close_reset(handle, close_cb));
```

Reply:
@santigimeno thanks for this suggestion.  
As I have stated previously  I'm fairly new to git and github and am not sure of how libuv maintainers want the commits.  Should I use github to "commit suggestion" - seems that is not really favored as it's commit log is not per CONTRIBUTING.md.  And in this instance won't  include the rest of the changes...  So in this case, I obviously continue to work in my workspace.   Should I simply add a new commit or do I `git commit --amend` again and force push to keep everything in one changeset?  Somewhat concerned that yesterday I got several build failures - though not from these changes it appears.

Reply:
OK, thanks.  I shall commit new changes rather than emend then.  Cheers.